src/Maker/MakeEntity.php

```diff
@@ -363,6 +371,21 @@ private function askForNextField(ConsoleStyle $io, array $fields, string $entity
             $data['nullable'] = true;
         }
 
+        if ($io->confirm('Do you want to add validation constraints to the field', false)) {
+            $isFirstConstraint = true;
+
+            while (true) {
+                $constraint = $this->askValidationConstraint($io, $isFirstConstraint);
+                $isFirstConstraint = false;
+
+                if (null === $constraint) {
+                    break;
+                }
```

Comment:
Why not `while ($constraint = $this->askValidationConstraint($io, $isFirstConstraint))` ?